### PR TITLE
Fix compiling S-Function with Matlab R2018b

### PR DESCRIPTION
**Pull request by diegoferigo**
This is a temporary fix of https://github.com/robotology/blockfactory/issues/4. A more stable fix should be found.

Closes https://github.com/robotology/blockfactory/issues/4.

### CODE REVIEW

File: sources/Simulink/CMakeLists.txt [hunk 1]
```diff
@@ -25,6 +25,9 @@ if(CMAKE_CXX_COMPILER_ID MATCHES "GNU")
     target_compile_options(Simulink PRIVATE -Wno-format-overflow)
 endif()
 
+# Fix https://github.com/robotology/blockfactory/issues/4
+target_compile_options(Simulink PRIVATE "-fvisibility=default")
+
```
[the remark this answers is not in the document]
Comment (diegoferigo) [reply sources/Simulink/CMakeLists.txt]: Unfortunately it is not that easy, have a look at https://gitlab.kitware.com/cmake/cmake/blob/master/Modules/FindMatlab.cmake#L1021-1098.

I'm actually considering to include in this project a vendored version of `FindMatlab.cmake`, what's your opinion?

[the remark this answers is not in the document]
Comment (traversaro) [reply sources/Simulink/CMakeLists.txt]: > I'm actually considering to include in this project a vendored version of `FindMatlab.cmake`, what's your opinion?

Ok for me.

[the remark this answers is not in the document]
Comment (traversaro) [reply sources/Simulink/CMakeLists.txt]: > Unfortunately it is not that easy, have a look at https://gitlab.kitware.com/cmake/cmake/blob/master/Modules/FindMatlab.cmake#L1021-1098.

Ok, but do you agree that in the current form the PR is breaking MSVC builds?

[the remark this answers is not in the document]
Comment (diegoferigo) [reply sources/Simulink/CMakeLists.txt]: Yes it does, but as usual CI cannot catch it since there's no Matlab / Simulink support.

**Pull request merged**